docs/docsite/rst/os_guide/intro_zos.rst

```diff
@@ -0,0 +1,247 @@
+    * block or task - ``environment`` key word.
+
+For more details, see :ref:`playbooks_environment`.
+
+Configure the remote Python interpreter
+---------------------------------------
+
+Ansible requires a Python interpreter to run most modules on the remote host, and it checks for Python at the 'default' path ``/usr/bin/python``.
+
+On z/OS UNIX, the Python3 interpreter (from `IBM Open Enterprise SDK for Python <https://www.ibm.com/products/open-enterprise-python-zos>`_)
+is often installed to a different path, typically something like: ``/usr/lpp/cyp/v3r12/pyz``.
+
+The path to the Python interpreter can be configured with the Ansible inventory variable ``ansible_python_interpreter``.
+For example:
+
+.. code-block:: ini
+
+    zos1 ansible_python_interpreter:/usr/lpp/cyp/v3r12/pyz
+
+When the path to the Python interpreter is not found in the default location on the target host,
+an error containing the following message may result: ``/usr/bin/python: FSUM7351 not found``
+
+For more details, see: :ref:`python_interpreters`.
+
+Configure the remote shell
+--------------------------
+The z/OS UNIX System Services managed node includes several shells.
+Currently the only supported shell is the z/OS Shell located in path ``/bin/sh``.
+To configure which shell the Ansible control node uses on the target node, set inventory variable
+:ref:`ansible_shell_executable<ansible_shell_executable>`. For example:
+
+.. code-block:: ini
+
+    zos1 ansible_shell_executable=/bin/sh
+
+Enable Ansible pipelining
+-------------------------
+Enable :ref:`ANSIBLE_PIPELINING` in the ansible.cfg file.
+
+When Ansible pipelining is enabled, Ansible passes any module code to the remote target node
+through Python's stdin pipe and runs it in all in a single call rather than copying data to temporary files first and then reading from those files.
+For more details on pipelining, see: :ref:`flow_pipelining`.
+
+Enabling this behavior is encouraged because Python will tag its pipes with the proper encoding, so there is less chance of encountering encoding errors.
+Further, using Python stdin pipes is more performant than file I/O.
+
+
+Include the following in the environment for any tasks performed on z/OS UNIX managed nodes.
+
+.. code-block:: yaml
+
+    PYTHONSTDINENCODING: "cp1047"
+
+When Ansible pipelining is enabled but the ``PYTHONSTDINENCODING`` property is not correctly set, the following error may result.
+Note, the hex ``'\x81'`` below may vary depending on the source causing the error:
+
+.. code-block::
+
+    SyntaxError: Non-UTF-8 code starting with '\\x81' in file <stdin> on line 1, but no encoding declared; see https://peps.python.org/pep-0263/ for details
+
+
+Unreadable characters
+---------------------
+
+Seeing unreadable characters in playbook output is most typically an EBCDIC encoding mix up.
+Double check that the remote environment is set up properly.
+Also check the expected file encodings, both on the remote node and the controller.
```

Comment:
```suggestion
Also check the expected file encodings, both on the remote node and the control node.
```

Reply:
So we use suggested edits a lot in our reviews. If you actually add them to batch and commit (or in this case, just commit the suggestion) it will resolve this comment for you.